Dockerfile

```diff
@@ -44,8 +44,12 @@ ENV SPRING_PROFILES_ACTIVE=prod
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y \
-    postgresql nginx openjdk-17-jre-headless supervisor git gettext-base
+    apt-get -y install wget gnupg nginx openjdk-17-jre-headless supervisor git gettext-base
```

Comment:
we can add a repo before doing a first `apt-get update`  and then just do the update once, can't we?
You'll be also able to simply add `postgresql-13` to the existing `apt-get install` list

Reply:
I'll still need to install `wget` and `gnupg` before in order to add the postgres repo